### DISCUSSION TITLE
Issue with the underline Z-Wave Driver  - Nice / Fibaro IBT4ZWAVE Device

### DISCUSSION THREAD

**Discussion by jameswardzala** [zwave-js/zwave-js#6511] [Request Support / Investigate Issue]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

What is happening?

The device has stopped functioning in the last 4 weeks, i presume due to an update 

What did you expect to happen instead?

cant open or close the gates any longer with this device

[z-ui_2023-11-21.log](https://github.com/zwave-js/node-zwave-js/files/13441266/z-ui_2023-11-21.log)

  2. Click on '...'
  3. Scroll down to '...'

[z-ui_2023-11-21.log](https://github.com/zwave-js/node-zwave-js/files/13441269/z-ui_2023-11-21.log)
[z-ui_2023-11-21.log](https://github.com/zwave-js/node-zwave-js/files/13441272/z-ui_2023-11-21.log)

Anything else we should know?

There is a new driver available i see but not for home assistant implementation

### Software versions

Driver (node-zwave-js): ...

Z-Wave JS UI: ... 3.0.2

Home Assistant Z-Wave Integration: ... 3.02

Home Assistant Z-Wave JS Addon: ... 9.3.1

ioBroker.zwave2 Adapter: ...

If you are using something non-standard, tell us here: ...

### Device information

Manufacturer: ... Figaro
Model name: ... IBT4ZWAVE
Node ID: ...008

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [X] The log includes a re-interview of the problematic device _(if applicable)_.

- [X] The log includes the problematic interaction with the device _(if applicable)_.

- [X] I provided the node ID of the problematic device _(if applicable)_.

### Upload Logfile

[z-ui_2023-11-21.log](https://github.com/zwave-js/node-zwave-js/files/13441294/z-ui_2023-11-21.log)

[a comment by an automated account is not shown]

**Comment by jameswardzala** [comment]
Yes I did this – the log should be the correct format.

**Comment by jameswardzala** [comment]
[zwave-js-ui-store.zip](https://github.com/zwave-js/node-zwave-js/files/13442495/zwave-js-ui-store.zip)
[nodes.json](https://github.com/zwave-js/node-zwave-js/files/13442496/nodes.json)
[node_8.json](https://github.com/zwave-js/node-zwave-js/files/13442498/node_8.json)

please see attached logs as requested.

**Comment by AlCalzone** [comment]
After trying to control the device, it sends back an Application Status report, which is not understood by Z-Wave JS, but typically means "can't do that". Other than that, the log does not contain any info as to why this doesn't work. The commands to the device are received and understood, but not executed.

Maybe the motor is jammed?

**Comment by jameswardzala** [comment]
Good Morning

[A screenshot of a computer  Description automatically generated]

The device keep coming up as unknown after I select 'open' for example

I don't seem to get anything in the log now apart from :

2023-11-23 10:42:57.349 INFO Z-WAVE: [Node 008] Value updated: 38-0-targetValue null => 99
2023-11-23 10:42:57.351 INFO Z-WAVE: [Node 008] Value updated: 38-0-currentValue null => 99
2023-11-23 10:43:02.392 INFO Z-WAVE: [Node 008] Value updated: 38-0-targetValue 99 => null
2023-11-23 10:43:02.394 INFO Z-WAVE: [Node 008] Value updated: 38-0-duration unknown => unknown
2023-11-23 10:43:02.396 INFO Z-WAVE: [Node 008] Value updated: 38-0-currentValue 99 => null


The motor is fine it opens and closes on the fob.

Any idea where I can go with this ? could it be down to the driver update ? as it stopped a couple of weeks ago ?

I see there is a new driver v9.3.2 but I cant see any way of updating this from home assistant ?

Thanks James.

**Reply by AlCalzone** [reply]
You cannot reply to this discussion via e-mail if you want to attach files or include images. You have to do it via the Web UI.

**Reply by AlCalzone** [reply]
Anyways, I don't see how this can be related to Z-Wave JS updates. The commands are standardized and can't be changed nilly-willy. What changed is how the device reacts to them. I'm not sure if the remote is also using Z-Wave commands?

You could try excluding, factory-reset and including it again.
